library/ssl_tls13_invasive.h

```diff
@@ -20,13 +20,13 @@
 
 #include "common.h"
 
-#if defined(MBEDTLS_PSA_CRYPTO_C)
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
```

Comment:
This should still be `PSA_CRYPTO_C` (see other comments).

Actually I didn't notice before, but this is a 1.3-only header file, and 1.3 has a compile-time dependency on `PSA_CRYPTO_C` so we shouldn't need a `#if` at all, or perhaps if we need one it should be `MBEDTLS_SSL_PROTO_TLS1_3` guarding the whole header file?